docs/usage.md

```diff
@@ -0,0 +1,194 @@
+### Intro to Kubernetes Usage
+
+Once Kubernetes is running and healthy you can interact with it via REST API or using the kubectl command line tool.
+
+- [Configure the Kubernetes Endpoint](#config)
+- [Create a Kubernetes Pod Definition](#pod)
+- [Create a Kubernetes Service Definition](#service)
+- [Launch a Kubernetes Pod via REST API](#api)
+- [Launch a Kubernetes Pod via kubectl](#kubectl)
+- [View Usage Metrics](#metrics)
+
+<a name="config"></a>
+#### Configure the Kubernetes Endpoint
+
+Export the Kubernetes URL to an environment variable, where `<hostname>` is the Mesos Master hostname:
```

Comment:
Better "to the 'KUBERNETES_MASTER' env var" because for kubectl the name is important.